src/Okta.Idx.Sdk.E2ETests/Drivers/WebDriverDriver.cs

```diff
@@ -0,0 +1,31 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+
+namespace Okta.Idx.Sdk.E2ETests.Drivers
+{
+    public class WebDriverDriver : IDisposable
+    {
+        private Lazy<IWebDriver> _webDriver = new(() => SetupWebDriver());
+
+        public IWebDriver WebDriver => _webDriver.Value;
+
+        private static IWebDriver SetupWebDriver()
+        {
+            var options = new ChromeOptions();
+//            options.AddArgument("--headless");
```

Comment:
Should we remove this line?

Reply:
I think we may want to uncomment this line eventually, moving from local env to Travis CI.

Reply:
removing the line, we know where to put it when we need it